CONTRIBUTING.md

```diff
@@ -329,6 +329,83 @@ If you already have the `refinebio-examples` docker image:
+These types of PRs should only involve well-polished and ready for the public material.
+
+**Scenario 1: All changes from staging should be made live**  
+
+- Create a new branch from the most up-to-date `master` branch.
+- Checkout your newly created branch.
+- If you are certain that all changes in `staging` should be carried over to `master`, then you can do a `git merge staging` into your new branch.
+- Now you can use the new branch to create the pull request to master as you would normally do.
+- Try to be as specific as possible about what PRs (and by relation, their commits) you are requesting to merge to `master`.
+
+This strategy allows you to resolve any merge conflicts in your new branch so we don't do a bad thing by committing conflict resolutions directly to `staging`.
+It also provides us with a "snapshot" if merges are continuing to happen to `staging` on other PRs -- this can make review hard if it keeps changing.
+
+**Scenario 2: Only some changes from staging should be made live**  
+
+- Create a new branch from the most up-to-date `master` branch.
+- Checkout your newly created branch.
+- For each commit that needs to be made live, add it to your new branch by using `git cherry-pick <commit_id>`.
+Or in GitKraken, you can right click on the commit and choose `Cherry pick commit`.
+- Now you can use the new branch to create the pull request to `master` as you would normally do.
+- Try to be as specific as possible about what PRs (and by relation, their commits) you are requesting to merge to `master`.
+
+This strategy allows us to move forward changes from `staging` that are ready to be public facing even if other changes aren't ready (or if there isn't great timing for when the other changes will be ready).
+
+_Tips for getting commits ids_  
+GitKraken shows commits, but sometimes I find it hard to follow which commits belong to which branch.
+If you checkout `staging` and use `git log` you can see all the most recent commits for the `staging` branch.
+If you want to save it to a file for easy browsing and copy/pasting you can use this command `git log --pretty=format:'%h was %an, %ar, message: %s' > git.log`
+
+#### Make it live, but quickly: direct merges to master, hotfixes
+
+In (hopefully rare) scenarios where something that has already been published is noted to be broken and should be addressed quickly, [hotfix branch](https://nvie.com/posts/a-successful-git-branching-model/#hotfix-branches) pull requests are allowed.
+These PRs should only be fairly small PRs and not anything that would require intense review.
+This more for situations where "this is broken and here's a fix".
+
+- Create a new branch from the most up-to-date `master` branch.
+- Checkout your newly created branch.
+- Make the hotfix change.
+- Create the pull request to `master` as you would normally do.
+- After your PR to `master` is approved, file a second PR from the same branch to `staging`.
```

Comment:
Probably worth adding that the next step (or maybe even before filing the second PR) will be merging `staging` into this branch. BUT, this will cause a problem if the merge to `master` has not yet happened. So maybe this should say:
> After your PR to `master` is approved and merged...